src/plugins/mer/meremulatordevice.cpp

```diff
@@ -684,6 +709,10 @@ void MerEmulatorDeviceManager::onDeviceListReplaced()
         if (!merEmulator)
             continue;
 
+        // Force emulator VM video settings update for edited emulator
+        if (merEmulator->id() == m_deviceUnderEdit)
+            merEmulator.constCast<MerEmulatorDevice>()->setDeviceModel(merEmulator->deviceModel());
+
```

Comment:
To me it seems this was already done during `MerEmulatorModeDialog` execution.

Reply:
Yes, but there may be the case when `MerEmulatorModeDialog` not called. For example: user edited device model (currently selected for emulator device) and apply changes, then we need to call `updateDconfDb()` and `scheduleSetVideoMode()` to update emulator VM settings.

Reply:
Then I think
1. it is not related to this commit, should be part of the commit that adds support for used defined device models
2. it is not necessary to call it on deviceListReplaced(), because only the device model name is stored with MerEmulatorDevice (the properties associated with the device model are applied directly to the VM)
3. we should ensure that all emulator devices have their properties up to date, so we should handle MerSdk::deviceModelsChanged instead and call setDeviceModel on all emulators.
3.1 Possible to limit that by device models that really changed?

Reply:
Reworked. Refer to `MerEmulatorDeviceManager::onDeviceModelsChanged`.